tests/fixtures/datacite_jpal_record_full.xml

```diff
@@ -0,0 +1,96 @@
+<?xml version="1.0" encoding="UTF-8"?>
```

Comment:
I would consider putting these fixtures in a "datacite" subfolder and renaming them to just "jpal_record...". I think it will be more clear if the fixtures subfolders match our sources folder (especially as we add potentially a lot more source transforms over time).